types/aws-lambda/common/cloudfront.d.ts

```diff
@@ -60,9 +60,9 @@ export interface CloudFrontEvent {
     config: {
         readonly distributionDomainName: string;
         readonly distributionId: string;
-    } & (
-        | { readonly eventType: 'origin-request' | 'origin-response' }
-        | { readonly eventType: 'viewer-request' | 'viewer-response'; readonly requestId: string });
+        readonly eventType: 'origin-request' | 'origin-response'|'viewer-request' | 'viewer-response';
```

Comment:
completely trivial nit: spacing around the middle `|`

Reply:
Indeed, sorry... I am able to fix this before the merge ?

Reply:
It may require me to re-approve, but that's no problem.

Reply:
I made a commit for this : https://github.com/nhodin/DefinitelyTyped/commit/6564ad32b7a2e1a1eed06cb419b3cf26582bc970 . Should i make a new merge request ?